docs/reference/method/MongoDBGridFSBucket-openDownloadStream.txt

```diff
@@ -36,7 +36,28 @@ Errors/Exceptions
 .. include:: /includes/extracts/error-gridfs-filenotfoundexception.rst
 .. include:: /includes/extracts/error-driver-runtimeexception.rst
 
-.. todo: add examples
+Examples
+--------
+
+.. code-block:: php
+
+   <?php
+
+   $bucket = (new MongoDB\Client)->test->selectGridFSBucket();
+
+   $stream = fopen('php://temp', 'w+b');
+   fwrite($stream, "foobar");
+   rewind($stream);
+
+   $id = $bucket->uploadFromStream('filename', $stream);
+
+   $bucket->openDownloadStream($id);
+
+   var_dump(stream_get_contents($stream, -1, 0));
```

Comment:
Since the download stream will point to the beginning of the file, we can just use `stream_get_contents($downloadStream)` here (or whatever you call the variable).